docs/use_cases/personalized_search.md

```diff
@@ -75,21 +73,21 @@ user_preference_weight = 0.3
 biased_query_embedding = query_weight * query_embedding + user_preference_weight * user_preference_vector
 ```
 
```

Comment:
Make changes per comments in my previous review:

line 37 (above):
use singular for heading of a table:
instead of "Key Components" "Descriptions"
say "Key Component" "Description"